package.json

```diff
@@ -371,7 +371,8 @@
         "opn": "^5.1.0",
         "request": "^2.83.0",
         "simple-git": "^1.77.0",
-        "vscode-extension-telemetry": "^0.0.6"
+        "vscode-extension-telemetry": "^0.0.6",
+        "vscode-azure-appservice": "https://github.com/EricJizbaMSFT/vscode-azure-sdk/releases/download/appservice-v0.1.1/vscode-azure-appservice-0.1.1.tgz"
```

Comment:
Will you switch to the "official" repo with this check-in?

Reply:
Yes - I won't merge these PRs until the official repo is created, reviewed, and I've published the packages to npmjs.